templates/razorpay-settings-templates.php

```diff
@@ -156,6 +156,42 @@ function displayPaymentAction()
         echo $paymentAction;
     }
 
+        /**
+     * Currency field of settings page
+    **/
+    function displayCurrencyAction()
+    {
+        $default = get_option('currency_action_field');
+
+        $supported_currencies = json_decode(file_get_contents(__DIR__."/../supported-currencies.json"), true)['supported-currencies'];
+
+        $supported_currency_codes = [];
+
+        foreach ( $supported_currencies as $k => $v )
+        {
+            array_push($supported_currency_codes, $v['ISO Code']);
+        }
+
+        $currency= '<select name="currency_action_field" id="currency_action"> ';
+                    
+        foreach ($supported_currency_codes as $index => $value)
+        { 
+            if ($value === $default)
+            {
+                $currency .='<option value="' . $value . '" selected="selected"'; 
+            }                        
+            else
+            {
+                $currency .='<option value="' . $value . '"'; 
+            }
+            $currency .= '>' . $value . '</option>';
```

Comment:
close option in one line

Reply:
fixed here https://github.com/razorpay/razorpay-quick-payments/blob/b7757b7948021cd22c1eb9d5326e5ea8c3decb1b/templates/razorpay-settings-templates.php#L191